.travis.yml

```diff
@@ -63,4 +64,6 @@ script:
   - make
   - make check
   - make deb
+  - docker build -f docker/Dockerfile -t centos:7 .
+  - docker run --rm -v $PWD:/home centos:7 make docker-rpm
```

Comment:
Using build matrix is already part of [#162](https://github.com/sociomantic-tsunami/libdrizzle-redux/pull/162/files#diff-354f30a63fb0907d4ad57269548329e3R51)